docs/Makefile

```diff
@@ -10,9 +10,21 @@ BUILDDIR      = build
 endif
 SOURCEDIR	  = .
 #IMAGEDIRS can be a list of directories that contain SVG files and are relative to the SOURCEDIR
-IMAGEDIRS     = _static
+IMAGEDIRS     = _static model_developers/images
 
 # SVG to PDF conversion
+# support inkscape versions <1.0 and >1.0
+# https://wiki.inkscape.org/wiki/Using_the_Command_Line#Deprecations_and_Replacements
+ifeq ($(shell inkscape --version | cut -d " " -f 2 | cut -d "." -f 1),0)
+SVG2PDF_FILE = -f
+SVG2PDF_PDF = -A
+SVG2PDF_PNG = -e
+else
+SVG2PDF_FILE =
+SVG2PDF_PDF = --export-type=pdf -o
+SVG2PDF_PNG = --export-type=png -o
+endif
```

Comment:
This will make my life easier.

Reply:
If we apply this, we should close https://github.com/inmanta/inmanta-core/issues/2534 as well.